packages/frontend/tailwind.config.js

```diff
@@ -7,6 +7,9 @@ export default {
       "tennis-green": "#2ecc71",
       "squash-yellow": "#f1c40f",
       "font-gray": "#1c3e50",
+      "font-white": "#ffffff",
+      "pg-bar-blue": "#3498db",
+      "pg-bar-gray": "#95A5A6",
```

Comment:
Remove these two lines, rebase off `main`, and use `progress-blue` and `p-grey-200` instead.

```suggestion
```

P.S. I like your use of American spellings with colour names — I think that should be a standard, lol.